Fix some out of date Solarisism's that ended up in upstream man page.

Need to contribute upstream.

--- dbus-glib-0.102/doc/dbus-binding-tool.1
+++ dbus-glib-0.102/doc/dbus-binding-tool.1
@@ -1,5 +1,5 @@
 '\" te
-.TH dbus-binding-tool 1 "26 Feb 2009" "SunOS 5.11" "User Commands"
+.TH dbus-binding-tool 1 "3 Nov 2016" "Solaris 11.4" "User Commands"
 .SH "NAME"
 dbus-binding-tool \- C language GLib bindings generation utility\&.
 .SH "SYNOPSIS"
@@ -264,20 +264,6 @@
 .sp
 .sp 1
 .in -32n
-.SH "ATTRIBUTES"
-.PP
-See \fBattributes\fR(5)
-for descriptions of the following attributes:
-.sp
-.TS
-tab() allbox;
-cw(2.750000i)| cw(2.750000i)
-lw(2.750000i)| lw(2.750000i).
-ATTRIBUTE TYPEATTRIBUTE VALUE
-AvailabilitySUNWdbus-bindings
-Interface stabilityVolatile
-.TE
-.sp
 .SH "SEE ALSO"
 .PP
 \fBdbus-cleanup-sockets\fR(1),
@@ -286,7 +272,7 @@
 \fBdbus-send\fR(1),
 \fBdbus-uuidgen\fR(1),
-\fBlibdbus-glib-1\fR(3),
-\fBattributes\fR(5)
+\fBlibdbus-glib-1\fR(3lib),
+\fBattributes\fR(7)
 .SH "NOTES"
 .PP
 Written by Brian Cameron, Sun Microsystems Inc\&., 2009\&.
